bowtie/_cli.py

```diff
@@ -1271,6 +1298,179 @@ def validate(
     return asyncio.run(_run(fail_fast=False, **kwargs, cases=[case]))
 
 
+@subcommand
+@IMPLEMENTATION
+@dialect_option()
+@format_option()
+@click.option(
+    "--runs",
+    "-r",
+    "runs",
+    type=int,
+    default=3,
+    show_default=True,
+    help=("Number of runs used to run benchmarks."),
```

Comment:
Parens aren't needed in all these, they're just needed if you want multiple lines.